### ISSUE TITLE
Lesson pilot with GitLab 25th and 26th of November @TNO 

### ISSUE THREAD

**Issue by svenvanderburg** [carpentries-incubator/python-intermediate-development#404]
We will teach the lesson using GitLab on 25th & 26th of November. We organize the workshop for [TNO](https://www.tno.nl/nl/) and they use GitLab primarily. 

We will fork the material and adapt it for GitLab. Here is [the fork](https://github.com/esciencecenter-digital-skills/python-intermediate-development-gitlab) in case anyone is interested. Is there any interest in contributing back the GitLab variant, for example using different tabs showing either GitHub or GitLab instructions?

#find-PR-and-replace-MR

**Comment by svenvanderburg** [comment]
The workshop went well and we would be happy to contribute back the GitLab material (or leave this here as a reference).

**Comment by anenadic** [comment]
How many participants did you have @svenvanderburg? Maybe open an issue with GitLab material reference so people can find it (we can leave the issue open).

**Comment by svenvanderburg** [comment]
Good idea, I opened a separate issue here: https://github.com/carpentries-incubator/python-intermediate-development/issues/425

We had about 15 participants
